Library/Formula/beanstalk.rb

```diff
@@ -12,6 +10,7 @@ class Beanstalk < Formula
   end
 
   def install
+    mkdir_p var/"log"
```

Comment:
:+1: - Will yank a bottle off the bot and take a peek, or bottle something locally and check. Whichever proves quicker.